Create FPKI Stat Report

weirdscience opened the pull request:
Add a new useful tools page for the FPKIMA Statistical Report

Commit:
Create FPKI Stat Report

Add a new useful tools page for the FPKIMA Statistical Report

Commit:
Clean-up and add 12 months

Cleaned the markdown down, added line breaks, and added 12 months of availability info

Commit:
Fixed spelling

Found some misspelled words.

lachellel commented on _tools/fpki-stat-report:
```diff
@@ -0,0 +1,113 @@
+# The Federal PKI Management Authority Statisical Report
```
Needs to be a markdown file (.md)
Needs the header info

---
layout: default 
title: 
collection: tools
permalink: tools/<linklocation>/
---

weirdscience replied:
Updated.

lachellel commented on _tools/fpki-stat-report:
```diff
@@ -0,0 +1,113 @@
+
+## FPKI Affiliate Status Summary
+
+The following table summarizes the Scorecard status of each connected CA to Federal Common Policy or Federal Bridge.
```
I'd clarify this statement?  It's always been a bit confusing that only the CAs that are directly signed by Common or Federal Bridge CA are listed.   The table doesn't list CAs either?

weirdscience replied:
Updated. Should only be a status of affiliates and not CAs.

lachellel commented on _tools/fpki-stat-report:
```diff
@@ -0,0 +1,113 @@
+| --------- | ---------- | ---------- | ------ | ------ |
+| No Activity |
+
+The following certificates have been removed from the FPKI Trust Infrastructure in the last 30 days.
```
Is this supposed to be inclusive of all CAs? Or only the ones that Common or the Federal Bridge CA sign?

weirdscience replied:
Just MA CAs because we don't have a good method to identify all CAs yet. Future enhancement to include all CAs.

lachellel commented on _tools/fpki-stat-report:
```diff
@@ -0,0 +1,113 @@
+| GSA ACES | IdenTrust ACES CA 2 | Federal Bridge CA 2016 | 46 68 3C 1B 51 B5 1C B9 1F 73 97 74 35 E4 3C F8 2B E9 CC 74 | 09/01/2018 |
+| IdenTrust NFI | IdenTrust Global Common Root CA 1 | Federal Bridge CA 2016 | 66 FC 51 BF 22 5B D8 C4 E6 11 E1 56 E7 30 A4 83 FF 99 90 C3 | 09/25/2018 |
+
+### Repository Availability Reporting
```
Same suggestion as above - this has been confusing since the table doesn't list all the CAs.  It's only the CAs that are directly signed by Federal Bridge CA or Common?

weirdscience replied:
Same comment as above. The intent is to alert readers that the certificates in this specific table may be re-issued and to watch out for a new CA cert.

lachellel commented:
@weirdscience I added some comments.  The first comment is the reason the file isn't rendering in the build! 

https://federalist-proxy.app.cloud.gov/preview/gsa/fpki-guides/fpki-stat-report/

 One option you might _want_ to consider _in the future_ is using _data_ files for all of the sub-tables? Then the data files can be updated with the new stats or changes, and the markdown and display doesn't need to be changed.  An example is used in System Notifications.

Commit:
Update to address comments.

Update to address comments.

weirdscience reviewed (commented):
Updated.

weirdscience commented:
Roger that @lachellel. I've updated based on your comments. 

I have a sprint planned to automate each section and using a similar format as the system notification is a great idea. I'm still working on a vision for a final product that incorporate the three sections (compliance status, certificate activity, and repo availability) in a single dashboard. A final product that incorporates everything in an automated fashion is the goal.

Commit:
Fix Permalink

Fix permalink

weirdscience reviewed (commented):
fix permalink

lachellel commented:
are you ready for a merge?

weirdscience commented:
:+1:

Commit:
Missed Slash

This should do it.

weirdscience commented:
@lachellel ready to merge.

lachellel committed:
more formatting updates

The page name was displayed twice. 
Included mailto: for the emails to direct to the right client (and remove the 404)
Added the links to the top for navigation
Updated the headers to have the same format and remove duplicative top header with sub-header right underneath
Moved the legend and fixed the formatting
Tried to update for action oriented verbs per plain language guidance

@weirdscience

lachellel commented:
@weirdscience take a look at my last commit.  (https://github.com/GSA/fpki-guides/pull/233/commits/0e93f608c925b79264be97d426888fffc5956def)

lachellel committed:
minor update

updating the fpki graph title to match the activity report
consistency
